app/javascript/controllers/contributors_controller.js

```diff
@@ -0,0 +1,44 @@
+import { Controller } from "stimulus"
+
```

Comment:
Note that this is not an entirely new controller - it was renamed from `contributor_controller.js` to `contributors_controller.js` and also changed a bit.

Reply:
soon we will document why this is so